service/src/main/java/com/basis/grupoum/sgt/service/Dominio/Usuario.java

```diff
@@ -0,0 +1,37 @@
+package com.basis.grupoum.sgt.service.Dominio;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+import lombok.Generated;
+import lombok.Setter;
+import lombok.Getter;
+
+import javax.persistence.*;
+
+@Setter
+@Getter
+@Entity
+@Table(name = "tb_usuario")
+public class Usuario implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq_usuario")
+    @SequenceGenerator(name = "seq_usuario", allocationSize = 1, sequenceName = "seq_usuario")
+    @Column(name = "id")
+    private long id;
```

Comment:
Caso for usar esse diretório aqui, corrigir esse long para Long, a diferença é que minúsculo é o primitivo e maiúsculo é da Classe Long, que possui todos os métodos auxiliares pra manipulação, trocar o long por Long